mix.exs

```diff
@@ -3,7 +3,7 @@ defmodule Crutches.Mixfile do
 
   def project do
     [app: :crutches,
-     version: "2.0.0",
+     version: "3.0.0",
```

Comment:
I think we should keep it at 2.0.0, because 

1. Reduce churn
2. We haven't pushed 2.0.0 to hex, so it's not like the previous changes to `List.shorten` were released.